CDCgov/prime-reportstream
# Experience-7891: Disable organization-specific fetches as admin

This changeset disables a few Organization-specific requests to mitigate the number of 404s we're getting as admins try to fetch Organization resources:
- Organization settings
- Deliveries
- Submissions

There's not a one-size-fits-all solution here given the different fetch mechanisms we already have and how the data is rendered, so I tried to add minimal changes to prevent disrupting anything down the line.  I think going forward, we can make a generic `useOrganizationQuery` hook (or whatever) that'll automatically be disabled if the user is logged in as an admin without impersonating an Organization.  There are a lot of layers with our fetching that in my opinion should be untangled, but that's out of the scope of this pull request.

Test Steps:
1. Log in as an admin
2. Open the network tab
3. Without impersonating an organization:
  a. Navigate to `/submissions` --> ensure no GET request is made for PrimeAdmins Organization settings or submissions 
  b. Navigate to `/daily-data` --> ensure no GET request is made for for PrimeAdmins Organization settings or deliveries
4. While impersonating an organization:
  a. Navigate to `/submissions` --> ensure GET requests are made for selected Organization settings and submissions
  b. Navigate to `/daily-data` --> ensure GET requests are made for selected Organization settings and deliveries

## Changes

Added a small visual change for the pages to account for this case:

<img width="1480" alt="Screenshot 2023-01-18 at 17 09 47" src="https://user-images.githubusercontent.com/2421042/213306617-aa3e006a-f51a-4c2f-8828-8699b55ffcc7.png">
<img width="1480" alt="Screenshot 2023-01-18 at 17 09 49" src="https://user-images.githubusercontent.com/2421042/213306620-bdcb9f33-6b0a-4b3e-9f63-3db5c1f28897.png">

Valid user behavior is unaffected:
<img width="1480" alt="Screenshot 2023-01-18 at 17 09 54" src="https://user-images.githubusercontent.com/2421042/213306622-85fe9db9-7dfd-4f8f-9f00-c83908b33c01.png">
<img width="1480" alt="Screenshot 2023-01-18 at 17 09 59" src="https://user-images.githubusercontent.com/2421042/213306624-8cbed1a4-2f64-42fd-a376-28e2ff4aca49.png">


## Checklist

### Testing
<strike>- [ ] Ran `./prime test` or `./gradlew testSmoke` against local Docker ReportStream container?</strike>

## Linked Issues
- Fixes #7891 
- Fixes #7890
